### PR TITLE
frontend: `MangaCore`

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: src/app.postcss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 @import "highlight.js/scss/github-dark.scss";
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```
Comment (baseplate-admin) [inline src/app.postcss:5]: Why is this like this?

File: src/routes/manga/+layout.svelte [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                            <a
+                                href="/user/login"
+                                class="btn rounded-[0.4rem] bg-primary-500 px-5 py-4 text-base font-semibold leading-none md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+                            >
+                                Login
+                            </a>
+                        </auth-buttons>
+                    {/if}
+                {/if}
+            </navbar>
```
Comment (baseplate-admin) [inline src/routes/manga/+layout.svelte:127]: This is repeated logic no? Use a named layout

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/routes/manga/+layout.svelte (hunk 2 above)
Comment (tokitou-san) [reply src/routes/manga/+layout.svelte:127]: yea I think so

Comment (baseplate-admin) [reply src/routes/manga/+layout.svelte]: Lttm

File: src/routes/manga/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                    class="btn btn-icon absolute -right-[1vw] top-1/2 -translate-y-1/2 z-20 hidden h-[2.25vw] w-[2.25vw] rounded-[0.375vw] bg-secondary-800 md:flex"
+                >
+                    <Chevron
+                        color="text-white"
+                        class="w-[1.25vw] -rotate-90"
+                    />
+                </button>
+            </slider-buttons>
+        </slider>
+    </landing>
```
Comment (baseplate-admin) [inline src/routes/manga/+page.svelte:107]: image

we shouldn't have this large thingy

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/routes/manga/+layout.svelte (hunk 2 above)
Comment (baseplate-admin) [reply src/routes/manga/+layout.svelte]: > but this structure is bit different so what do you think?

what do you mean by this actually?

File: src/app.postcss (hunk 1 above)
Comment (baseplate-admin) [reply src/app.postcss:5]: I am not a big fan of this approach tbh. Lets see if we can revert it

File: src/routes/manga/+page.svelte (hunk 3 above)
Comment (tokitou-san) [reply src/routes/manga/+page.svelte:107]: huh! what :\
It looks really cool, whats the issue?

Comment (baseplate-admin) [reply src/routes/manga/+page.svelte:107]: >  It looks really cool, whats the issue?

Not more animes. Add nhentai

Comment (tokitou-san) [reply src/routes/manga/+page.svelte:107]: > Not more animes. Add nhentai

Dont want this kinda section?
direct cards?

File: src/app.postcss (hunk 1 above)
Comment (tokitou-san) [reply src/app.postcss:5]: > I am not a big fan of this approach tbh. Lets see if we can revert it

Okay sure I will take a look, yea I think this isnt happening with other routes, anyways I dont like this approach either :\

Comment (baseplate-admin) [reply src/app.postcss:5]: > Okay sure I will take a look, yea I think this isnt happening with other routes, anyways I dont like this approach either :\

oh :\ you should have notified me this

**Pull request closed without merging**